frontend/src/pages/share/[id].tsx

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+
+import type { AxiosResponse } from 'axios';
+import type { GetServerSideProps } from 'next';
+import type { SharedURLData } from 'types/shared-url';
+
+const SharePage: React.FC = () => null;
+
+export const getServerSideProps: GetServerSideProps = async (ctx) => {
```

Comment:
- If the ID doesn't exist, could we redirect the user to a 404 page?
- Should you try/catch the request?

Reply:
I get a 500 when the ID doesn't exist.

![When loading http://localhost:3000/share/hello, the default Next.js 500 error page is shown.](https://user-images.githubusercontent.com/6073968/231153475-df5c332a-9d45-4626-8b4f-f61c70c279af.png)